pkg/webhooks/server.go

```diff
@@ -501,6 +501,7 @@ func (ws *WebhookServer) resourceValidation(request *v1beta1.AdmissionRequest) *
 	admissionRequestTimestamp := time.Now().Unix()
 	kind := request.Kind.Kind
 	policies := ws.pCache.GetPolicies(policycache.ValidateEnforce, kind, "")
+	policies = append(policies, ws.pCache.GetPolicies(policycache.ValidateAudit, kind, "")...)
```

Comment:
Audit policies are handled in:
```
	ws.auditHandler.Add(request.DeepCopy())
```

If there's any `ValidationFailureActionOverrides` policies, they should be added as `Enforce` policies in the policy cache. Otherwise the webhook processes the same policy twice which is unnecessary.